tests/test_full_library.py

```diff
@@ -231,25 +234,24 @@ def test_full_lib(tmpdir):
                 allDecl=masterDict,
             )
```

Comment:
```suggestion
                allDecl=masterDict,
                add_relative=True
            )
```

Now we hopefully just need to specify that this is a cython package and activate your new flag.